<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettingWid</class>
 <widget class="QDialog" name="AdvancedSettingWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>688</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>688</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdvancedSettingWid</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="LeftTitleWid" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>39</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>39</height>
      </size>
     </property>
     <widget class="QPushButton" name="CloseBtn">
      <property name="geometry">
       <rect>
        <x>647</x>
        <y>0</y>
        <width>39</width>
        <height>39</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="HintLab">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>22</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>22</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="SettingTitleLab">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>9</y>
        <width>133</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>133</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>133</width>
        <height>22</height>
       </size>
      </property>
      <property name="text">
       <string>Setting</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="CenterHLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="LeftWid" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="LeftWidVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="LeftBtnWid" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="LeftBtnWidVLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="VideoSettingBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Video</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AudioSettingBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Audio</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PushStreamBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Push Stream</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="HotkeyBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Hotkey</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="SettingStackedWid">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QScrollArea" name="VideoSettingScrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="VideoSettingPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>524</width>
           <height>421</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="VSGLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="AudioSettingPage">
        <layout class="QGridLayout" name="ASGLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QWidget" name="PushStreamPage">
        <layout class="QGridLayout" name="PSGLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QScrollArea" name="HotkeySettingScrollArea">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="HotkeyPage">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>524</width>
           <height>421</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QGridLayout" name="HKGLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="BottomHLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="BtnBottomWid" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>74</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>74</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="StackBottomWid" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>74</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>74</height>
        </size>
       </property>
       <widget class="QPushButton" name="SetBtn">
        <property name="geometry">
         <rect>
          <x>161</x>
          <y>15</y>
          <width>90</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="CancelBtn">
        <property name="geometry">
         <rect>
          <x>288</x>
          <y>15</y>
          <width>90</width>
          <height>35</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
